### PR TITLE
Update doc for db revisions and archi

### PR DESCRIPTION
**Pull request by pbougue**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -218,12 +218,65 @@ If the COTS was successfully sent and processed by Kirin, the http response 200
...
+Its role is to regularly publish polling jobs destined to Kirin-workers.
+
+There is only one of these on each platform.
+
+#### Kirin-worker
+
+> Alias 'worker'
+
+Its role is to poll an external location and check if new information were published.
+In that case it processes it then store the result in db and send the corresponding info the Kraken.
```
Comment (kadhikari) [inline readme.md]: typo: corresponding info the Kraken.  -> corresponding info to Kraken.

File: readme.md [hunk 2]
```diff
@@ -218,12 +218,65 @@ If the COTS was successfully sent and processed by Kirin, the http response 200
 
 ## Development
 
-To generate a new migration script for database (after an upgrade of the model.py file):
+
+### Alembic database revisions
+
+To generate a new revision script for database (after an upgrade of the model.py file):
```
Comment (nonifier) [inline readme.md]:
```suggestion
To generate a new database revision script (after an upgrade of the model.py file):
```

File: readme.md [hunk 3]
```diff
@@ -218,12 +218,65 @@ If the COTS was successfully sent and processed by Kirin, the http response 200
...
+
+Kirin is split in 4 separate "components", as seen in honcho's Procfile.
+
+#### Kirin-webservice
+
+> Alias 'kirin' or 'web'
+
+Its roles are:
+* display the `/status`.
+* provide a POST endpoint for each type of realtime provider accepted.
```
Comment (nonifier) [inline readme.md]:
```suggestion
* provide a POST endpoint for each type of accepted realtime provider .  
```

File: readme.md [hunk 4]
```diff
@@ -218,12 +218,65 @@ If the COTS was successfully sent and processed by Kirin, the http response 200
...
+There can be several of these (if they are behind a load-balancer).
+
+#### Kirin-background
+
+> Alias 'load_realtime'
+
+Its role is to provide all information available in db for a given provider in the rabbitmq queue, so
+that Kraken can restart fully aware of realtime.
+
+There can be several of these if load is important.
```
Comment (nonifier) [inline readme.md]:
```suggestion
There can be several of these if the load is important.
```

File: readme.md [hunk 5]
```diff
@@ -218,12 +218,65 @@ If the COTS was successfully sent and processed by Kirin, the http response 200
...
+
+Its role is to regularly publish polling jobs destined to Kirin-workers.
+
+There is only one of these on each platform.
+
+#### Kirin-worker
+
+> Alias 'worker'
+
+Its role is to poll an external location and check if new information were published.
```
Comment (nonifier) [inline readme.md]:
```suggestion
Its role is to poll an external location and check if new information was published.
```

File: readme.md [hunk 6]
```diff
@@ -218,12 +218,65 @@ If the COTS was successfully sent and processed by Kirin, the http response 200
...
+In that case worker processes it then stores the result in db and sends the corresponding info to Kraken.
```
Comment (nonifier) [inline readme.md]:
```suggestion
In that case, the worker processes it, stores the result in db and sends the corresponding info to Kraken.
```

File: readme.md [hunk 7]
```diff
@@ -218,12 +218,65 @@ If the COTS was successfully sent and processed by Kirin, the http response 200
...
+
+There can be several of these if load is important. At least one per provider polled is recommended.
```
Comment (nonifier) [inline readme.md]:
```suggestion
There can be several of these if the load is important. At least one per polled provider is recommended.
```

**Pull request merged**